variables.tf

```diff
@@ -45,19 +45,19 @@ variable "resource_keys" {
 }
 
 variable "cos_instance_name" {
-  description = "The name to give the cloud object storage instance that will be provisioned by this module. Only required if 'create_cos_instance' is true."
+  description = "The name for the IBM Cloud Object Storage instance provisioned by this module. Applies only if `create_cos_instance` is true."
   type        = string
   default     = null
 }
 
 variable "cos_location" {
-  description = "Location to provision the cloud object storage instance. Only used if 'create_cos_instance' is true."
+  description = "The location for the Object Storage instance. Applies only if `create_cos_instance` is true."
   type        = string
   default     = "global"
 }
 
 variable "cos_plan" {
-  description = "Plan to be used for creating cloud object storage instance. Only used if 'create_cos_instance' it true."
+  description = "Plan to be used for creating cloud object storage instance. Applies only if `create_cos_instance` is true."
```

Comment:
We should probably add the list of all available options to the description? `standard`, `lite` and `cos-one-rate-plan` are the only supported values

Reply:
I clearly missed this description. Tried to find all the other `cos_plan` vars and include the correct values.